server/src/test/java/org/opensearch/index/store/remote/filecache/FileCacheTests.java

```diff
@@ -245,6 +262,15 @@ public void testStats() {
 
     }
 
+    public void testCacheRestore() throws IOException {
+        NodeEnvironment.NodePath fileCacheNodePath = new NodeEnvironment.NodePath(path);
+        createFile("0", "test-index", "0", "test.0");
+        FileCache fileCache = createFileCache(MEGA_BYTES);
+        assert (fileCache.usage().usage() == 0);
```

Comment:
Prefer to follow the prevailing style of using assertEquals or assertThat for tests